data/databaseinsert.js

```diff
@@ -0,0 +1,107 @@
+var mysql = require('mysql');
+var mysqlConfig = require("../database/config.js")
+var data = require("./FECdata2.js")
+
+var connection = mysql.createConnection(mysqlConfig);
+
+function getRandom(arr, n) {
+  var result = new Array(n),
+      len = arr.length,
+      taken = new Array(len);
+  if (n > len)
+      throw new RangeError("getRandom: more elements taken than available");
+  while (n--) {
+      var x = Math.floor(Math.random() * len);
+      result[n] = arr[x in taken ? taken[x] : x];
+      taken[x] = --len in taken ? taken[len] : len;
+  }
+  return result;
+}
+
+function getRandomInt(max) {
+  return Math.floor(Math.random() * Math.floor(max));
+}
+
+connection.connect(function(err) {
+  if (err) throw err;
+  console.log("Connected!");
+  console.log(data);
+
+
+  for (let i = 0; i < data.data.length; i++){
+    let isClaimed= getRandomInt(2)
+    let sql = `INSERT INTO businesses (business, claimed) VALUES ("${data.data[i].name}", ${isClaimed});`
+    connection.query(sql, (err, result) => {
+      if (err) throw err; 
+      console.log(result)
+    }) 
+  }
+  
+  // for (let i = 1; i < data.data.length + 1; i++){
+  //   let randomNumberOfTags = getRandomInt(2) + 1;
+  //   let randomTags = getRandom(data.tagsdata, randomNumberOfTags)
+  //   console.log(i);
+  //   let sql = `INSERT INTO Categories (businessID, Tag1, Tag2, Tag3) VALUES (${i}, "${randomTags[0]}", "${randomTags[1] ? randomTags[1] : null}", "${randomTags[2] ? randomTags[2] : null}" );`
+  //   connection.query(sql, (err, result) => {
+  //     if (err) throw err;
+  //     console.log(result);
+  //   })
+  // }
```

Comment:
to remove? or still working on the commented out bit?

Reply:
also the 2 blocks below

Reply:
the console log was just to get a confirmation back that the insert was working. I wont be removing it until actual production, as it was always just act as a confirmation console.log